.travis.yml

```diff
@@ -47,6 +47,35 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray TEST_REQUIREMENTS="setproctitle hypothesis psutil"
+           PROJECT_URL=https://github.com/ray-project/ray.git
+           PYTEST_ARGS="--duration=10 -k test_mini or test_basic"
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install .
+        - pip install --upgrade -r dev-requirements.txt
+        - pip install tornado
```

Comment:
What is`tornado` for?

Reply:
I'm not sure, I was part of the `install` not specific to any third-party project so I'm assuming it's something that cloudpickle needs for its tests?

https://github.com/cloudpipe/cloudpickle/blob/b192238f4496e5997da749bb3b86daa4d10bbc2f/.travis.yml#L89